app/inputs/course_assessment_answer_multiple_response_input.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+class CourseAssessmentAnswerMultipleResponseInput < SimpleForm::Inputs::CollectionRadioButtonsInput
```

Comment:
yup. I'm not sure how to namespace it though, it can't be used other than the MRQ...